src/com/company/Main.java

```diff
@@ -3,6 +3,71 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        byte clientOS = 0;
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 0) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+
+        byte clientOS2 = 1;
+        short clientDeviceYear = 2013;
+        if (clientOS2 == 1) {
+            if (clientDeviceYear >= 2015) {
+                System.out.println("Установите версию приложения для iOS по ссылке");
+            } else System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS2 == 0) {
+            if (clientDeviceYear >= 2015) {
+                System.out.println("Установите версию приложения для Android по ссылке");
+            } else System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }
+
+        int year = 2021;
+        if (year % 400 == 0) {
+            System.out.println(year + " год является високосным");
+        } else {
+            if (year % 4 == 0 && year % 100 != 0) {
+                System.out.println(year + " год  является високосным");
+            }
+            else System.out.println(year + " год не является високосным");
+        }
+
+
+        int deliveryDistance = 95;
+        if (deliveryDistance <= 20) {
+            System.out.println("Доставка в пределах 20 км занимает сутки");
+        } else if (deliveryDistance > 20 && deliveryDistance <= 60) {
```

Comment:
поскольку у нас один общий условный оператор, то данная ветка будет проверяться только в том случае, если условие выше было ложным. Отсюда получается, что первое условие будет уже автоматически выполняться
```suggestion
        } else if (deliveryDistance <= 60) {
```